src/gravel/controllers/nodes/systemdisk.py

```diff
@@ -84,7 +84,8 @@ class SystemDisk:
         "etc": "/etc",
         "logs": "/var/log",
         "etcd": "/var/lib/etcd",
-        "containers": "/var/lib/containers"
+        "containers": "/var/lib/containers",
+        "roothome": "/root"
```

Comment:
`/root` is needed for orch ssh config? or something else?

Reply:
yeah, ssh pubkey